src/manager.cpp

```diff
@@ -158,7 +159,15 @@ sdbusplus::async::task<bool>
         "rsync --compress --recursive --perms --group --owner --times --atimes"
         " --update --relative --delete --delete-missing-args "};
 
-    syncCmd.append(" "s + dataSyncCfg._path);
+    if (!srcPath.empty())
+    {
+        // Configure modified path as SRC path rsync if available.
```

Comment:
confusing the comment...

I think you meant like, use the given source path if available